### PR TITLE
Add sample to consistent with other languages.

### PR DESCRIPTION
**Pull request by ZejiaJiang**
# Description
Add sample to consistent with other languages
Addresses https://github.com/Azure/azure-sdk-for-java/issues/28906

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/servicebus/azure-messaging-servicebus/src/samples/README.md [hunk 1]
```diff
@@ -57,6 +57,9 @@ connection string value can be obtained by:
 ### Generate Shared Access Signature
 - [Generate SAS and receive message][ReceiveMessageAsyncUsingSasSample]
 
+### Advanced Configuration
+- [Advanced configuration sample set][AdvancedConfigurationSample]
```
Comment (ZejiaJiang) [inline sdk/servicebus/azure-messaging-servicebus/src/samples/README.md:61]: refer to .Net SDK and JS SDK, I add a AdvancedConfigurationSample class.

File: sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdvancedConfigurationSample.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        ServiceBusReceiverAsyncClient receiver = new ServiceBusClientBuilder()
+            .connectionString("<< CONNECTION STRING FOR THE SERVICE BUS NAMESPACE >>")
+            .receiver()
+            .queueName("<< QUEUE NAME >>")
+            .prefetchCount(100)
+            .buildAsyncClient();
+    }
+
+    /**
+     * Creates a session-enabled receiver that prefetch 100 messages.
```
Comment (ki1729) [inline sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdvancedConfigurationSample.java:45]: I thought session receiver sample is present to cover this scenario right? So maybe we can remove this one?

Comment (ZejiaJiang) [reply sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdvancedConfigurationSample.java:45]: Sure!

**Review by ki1729 — APPROVED** [review]
:shipit:

**Pull request merged**